kubernetes/nebula-graphd-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nebula
+  name: nebula-graphd
+spec:
+  selector:
+    matchLabels:
+      app: nebula
+  replicas: 3
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate  
+  template:
+    metadata:
+      labels:
+        app: nebula
+    spec:
+      containers:
+      - name: nebula-graphd
+        image: vesoft/nebula-graphd:nightly
+        imagePullPolicy: Always
+        ports:
+        - name: thrift
+          containerPort: 3699
+        - name: http
+          containerPort: 13000
+        - name: http2
+          containerPort: 13002
+        command: ["/usr/local/nebula/bin/nebula-graphd", "--flagfile=/usr/local/nebula/etc/nebula-graphd.conf", "--daemonize=false", "--meta_server_addrs=$(META_SERVER_ADDRS)" , "--ws_ip=0.0.0.0"]
```

Comment:
We could use `ConfigMap` to manage `nebula-graphd.conf`  file and mount it into container to overwrite default configuration file at `/usr/local/nebula/etc/nebula-graphd.conf`.